src/components/Book.js

```diff
@@ -0,0 +1,15 @@
+const Book = () => (
+  <>
+    <td>
+      Book Title 1
+    </td>
+    <td>
+      Progress 100%
+    </td>
+    <td>
+      <button type="button" className="btn btn-primary">REMOVE</button>
+    </td>
+  </>
+);
```

Comment:
- [ ] The `Book` component should take book data as a prop and renders the title and the author